sports_club/data/pos_config.xml

```diff
@@ -1,8 +1,20 @@
 <?xml version="1.0" encoding="utf-8"?>
 <odoo noupdate="1">
-
-    <record model="pos.config" id="pos_config_bar" forcecreate="0">
+    <record model="pos.config" id="point_of_sale.pos_config_main">
+        <field name="start_category" eval="1"/>
+        <field name="iface_available_categ_ids" eval="[Command.link(ref('pos_category_6')), Command.link(ref('pos_category_7')), Command.link(ref('pos_category_8')), Command.link(ref('pos_category_4'))]"/>
+        <field name="iface_start_categ_id" ref="pos_category_6"/>
+        <field name="active" eval="True"/>
```

Comment:
Thinking twice, I wonder if it is really needed. 
```suggestion
```

Reply:
I also gave a thought on this because active has default values as true
So it will be better to remove then.

Reply:
Ok so apparently it is useful, as the Shop is archived without that (sorry about that)